modules/k3s/src/test/java/org/testcontainers/k3s/Fabric8K3sContainerTest.java

```diff
@@ -34,7 +34,9 @@ public void shouldStartAndHaveListableNode() {
                 .withLogConsumer(new Slf4jLogConsumer(log))
             // }
         ) {
+            // starting_k3s {
```

Comment:
Maybe it is more aligned with other docs, if we remove the `start()` include and change the title of the code include to `Creating a K3sContainer`.

Reply:
Possibly, yes. I was really interested in the EC key problem. This was just an afterthought.